test/e2e/elasticsearch_test.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/openshift/elasticsearch-operator/pkg/utils"
 	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
unnecessary alias?

Reply:
it is.  this is from the go fmt which i see add it occationally